Sources/AppCoinsSDK/Domain/Entities/ClientWallet.swift

```diff
@@ -51,12 +51,12 @@ internal class ClientWallet: Wallet, Codable {
         }
     }
     
-    internal func getBalance(currency: Coin, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
+    internal func getBalance(currency: String, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
         if let address = self.address {
             transactionService.getBalance(wa: address) { result in
                 switch result {
                 case .success(let response):
-                    self.billingService.convertCurrency(money: String(response.usdBalance), fromCurrency: .USD, toCurrency: currency) {
+                    self.billingService.convertCurrency(money: String(response.usdBalance), fromCurrency: "USD", toCurrency: currency) {
```

Comment:
Considering that with the new request we can get the Wallet balance in the user's currency, we don't need to convert it after.

Reply:
Done